doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -743,7 +743,14 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
-  * Fixed an issue with the CFUN hooks when the Modem library is initialized during ``SYS_INIT`` at kernel level and makes calls to the :ref:`nrf_modem_at` interface before the application level initialization is done.
+  * Updated:
+
+    * Fixed an issue with the CFUN hooks when the Modem library is initialized during ``SYS_INIT`` at kernel level and makes calls to the :ref:`nrf_modem_at` interface before the application level initialization is done.
+    * Deprecated the Kconfig option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR`. Use the option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART` instead.
```

Comment:
Here we should refer to the uart trace snippet instead, to include device tree configuration.

Reply:
Why? That was introduced a while ago and it should be already documented somewhere else.

Reply:
True, though people using the formerly deprecated trace backend might change to `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART`, thinking that would be ok. Perhaps that could go under the removed section.

Reply:
Bu the deprecation of `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR` is a different change than the migration from a deprecated backend (`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ASYNC` or `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_SYNC`) to `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR`.

The change here is the deprecation of `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR`, for which the app needs to remove that entry from the `prj.conf` and use `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART` instead.
I have added an entry to the migration notes.

As for the removal of the deprecated entries, those should have been documented already; so no need to repeat that here.